user_interaction_txt_text.py

```diff
@@ -0,0 +1,94 @@
+import os
+from enum import Enum, auto
+from typing import Optional
+
+from user_file_manager import UserFileManager
+
+
+class MessageScope(Enum):
+    GLOBAL = auto()
+    ACTIVE = auto()
+    PASSIVE = auto()
+
+
+class UserInteraction():
+    def __init__(self, context: 'Context'):
+        self.context = context
+        self.user_names = self.context.user_names
```

Comment:
don\t like this approach. 
user_names - currently looks like public field of UserInteraction.
It can confuse the user of UserInteraction class, because it is not clear: who owns user names? Game or UserInteraction?

Reply:
Also, class Context does not contain field\property `user_names` as far as I remember

You should either add property `user_names` into Context or fetch it from Context in some other way

Reply:
Done